apps/mail/components/mail/mail-display.tsx

```
@@ -370,6 +370,347 @@ const MailDisplay = ({ emailData, index, totalEmails, demo }: Props) => {
+
+      // Wait for content to load, then print
+      printFrame.onload = function () {
+        setTimeout(() => {
+          try {
+            // Focus the iframe and print
+            printFrame.contentWindow.focus();
+            printFrame.contentWindow.print();
+
+            // Clean up - remove the iframe after a delay
+            setTimeout(() => {
+              if (printFrame && printFrame.parentNode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)